common/src/metrics.h

```diff
@@ -43,6 +45,30 @@ void get_csr_metrics(uint32_t *acc, uint32_t *refill, uint32_t *stall);
            stall_pre, stall_cnt, stall_cnt - stall_pre); \
   } while (0)
 
+#define CLK_OFF 0
+#define CS_OFF 1
+
+void get_flash_ticks(void);
+void set_flash_control(uint32_t val);
+
+#define FLASH_SETUP_METRICS \
+  set_flash_control(0x1)
```

Comment:
Instead of numeric constants 0x1, 0x0, 0x2 can you please use the constants from `generated/csr.h`? I guess this should be

    set_flash_control(1 << CSR_SPI_FLASH_COUNTER_CONTROL_ENABLE_OFFSET);

and similarly below. Or better yet, just use the generated helper functions:

    spi_flash_counter_control_enable_write(1);

It makes it clearer what is being set.

Reply:
Should be done now.